test/features/deploy_springboot_service.feature

```diff
@@ -84,4 +84,32 @@ Feature: Deploy spring boot service
       }
       """
 
-    Then GraphQL request on Data Index service returns ProcessInstances processName "orders" within 2 minutes
\ No newline at end of file
+    Then GraphQL request on Data Index service returns ProcessInstances processName "orders" within 2 minutes
+
+  @usertasks
+  Scenario: Deploy process-springboot-example service to complete user tasks
+    Given Kogito Operator is deployed
+    And Deploy springboot example service "process-springboot-example" with configuration:
+      | config | persistence | disabled |
+    And Kogito application "process-springboot-example" has 1 pods running within 10 minutes
+    And Service "process-springboot-example" contains 0 instance of process with name "orders" within 3 minutes
```

Comment:
This needs to be merged as done in https://github.com/kiegroup/kogito-cloud-operator/pull/324
we should wait for the merge of it before merging this one

Reply:
@Sgitario other PR has been merged, you can change it here :)

Reply:
changed